features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/customreaction/EmojiPicker.kt

```diff
@@ -87,15 +91,18 @@ fun EmojiPicker(
             val emojis = categories[category] ?: listOf()
             LazyVerticalGrid(
                 modifier = Modifier.fillMaxSize(),
-                columns = GridCells.Adaptive(minSize = 40.dp),
+                columns = GridCells.Adaptive(minSize = 60.dp),
```

Comment:
I agree this is way too large. Users want to see as many as Emoji as possible. On my phone it's currently 8 columns and it's OK for me. Also we do not have a way to filter Emoji (search), so better to see more Emojis.

Reply:
Maybe use `48` as it is the classical recommended touch area size?

Reply:
Out of curiosity, do we have some info now to search for emojis? Is `Emoji.label` intended for this, and is it localised?